### PR TITLE
Version 0.2.0-rc

### PR DESCRIPTION
**Pull request by OscartGiles**
### Overview

- Version 0.2.0-rc.0
- Update and refactor docs
- Single source of version number (pyproject.toml)

### CODE REVIEW

**Comment by Iain-S** [comment]
I'm getting 

```
ERROR    -  Error reading page 'post-processing.md': Snippet at path postprocess.py could not be found
```

when I try to serve the docs from the `docs/` directory.

File: docs/docs/cli_basics.md [hunk 1]
```diff
@@ -16,46 +16,26 @@ echo "I see a man in the dark standing against a light post. It seems to be in t
...
 ```
 
-=== "Python"
+We can break this down into the following components:
 
-```python
-import netts
+| CLI Command     | transcript.txt | outputs |
+| ----------- | -----|------ |
+| netts run   | Path to transcript   | path of output directory   |
```
Comment (Iain-S) [inline docs/docs/cli_basics.md:31]:
```suggestion
| CLI Command | transcript.txt | outputs |
| - | - | - |
| netts run | Path to transcript | path of output directory |
```

File: docs/mkdocs.yml [hunk 2]
```diff
@@ -2,28 +2,36 @@ theme:
...
-      - Overview: basic_usage.md
-      - Configuration: configuration.md
-      - CLI: cli.md
+      - CLI: cli_basics.md
+      - Python: python_basics.md
+      - Postprocess: post-processing.md
+      - CLI Configuration: configuration.md
+      - API:
+        - CLI: cli.md
+      # - CLI - details: cli.md
```
Comment (Iain-S) [inline docs/mkdocs.yml:36]: Delete?

File: netts/version.py [hunk 3]
```diff
@@ -1 +1,8 @@
-__version__ = "0.1.0"
+try:
+    from importlib.metadata import version  # type: ignore
+
+except ImportError:
+    from importlib_metadata import version
+
+
+__version__ = version("netts")
```
Comment (Iain-S) [inline netts/version.py:8]: Does this mean we only need to change it in pyproject.toml from now on?

**Review by Iain-S — CHANGES_REQUESTED** [review]

**Review by Iain-S — APPROVED** [review]

File: netts/version.py (hunk 3 above)
Comment (OscartGiles) [reply netts/version.py:8]: exactly

File: docs/docs/cli_basics.md (hunk 1 above)
Comment (OscartGiles) [reply docs/docs/cli_basics.md:31]: Done

File: docs/docs/python_basics.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+### Start the CoreNLP and OpenIE5 servers
+
+```python hl_lines="9-11"
+--8<-- "process_transcript.py"
+```
+
+Netts uses [Openie5](https://github.com/dair-iitd/OpenIE-standalone) and [CoreNLP](https://stanfordnlp.github.io/CoreNLP/) under the hood. These are both [Java](https://en.wikipedia.org/wiki/Java_(programming_language)) programmes that we installed in the previous step. We use a [context manager](https://book.pythontips.com/en/latest/context_managers.html) to start the servers, which makes sure they are both automatically shut down when processing finishes.
+
+!!! warning
+    The servers are extremely memory hungry. If the server fails to start you probably ran out of memory and failed silently. Try on a machine with more memory.
```
[the remark this answers is not in the document]
Comment (OscartGiles) [reply docs/docs/python_basics.md:39]: Done

File: docs/mkdocs.yml [hunk 5]
```diff
@@ -2,28 +2,36 @@ theme:
   name: material
   palette:
     primary: teal
-  logo: img/favicon_io/favicon-32x32.png
-  favicon: img/favicon_io/favicon.ico
+  logo: img/favicon_io/favicon.ico
+  # favicon: img/favicon_io/favicon.ico
```
[the remark this answers is not in the document]
Comment (OscartGiles) [reply docs/mkdocs.yml:6]: Ah yes didn't mean to leave it like that. Good spot.

**Comment by OscartGiles** [comment]
@Iain-S - made all your requested changes. Do you still have the issue with the missing snippet? Seems ok for me

**Comment by Iain-S** [comment]
@OscartGiles nope, you're all good to go. It was my fault for running `mkdocs` from the wrong directory.

**Pull request merged**